edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -448,15 +454,48 @@ internal static bool ExtractAgentTwinAndVerify(TwinCollection twinDesiredPropert
                 desiredProperties["schemaVersion"] = twinJobject["schemaVersion"];
                 desiredProperties["systemModules"] = twinJobject["systemModules"];
 
-                // Extract Integrity section
+                // Check if Manifest Trust Bundle is configured
+                X509Certificate2 manifestTrustBundleRootCertificate;
+                if (this.manifestTrustBundle.HasValue != true && twinJobject["integrity"] == null)
+                {
+                    throw new ManifestTrustBundleIsNotConfiguredException("Manifest Signing is Disabled.");
```

Comment:
Should probably log an error for each `throw`

Reply:
Yes thats a good point. Will do